evidence/pool.go

```diff
@@ -155,35 +154,21 @@ func (evpool *Pool) AddEvidence(ev types.Evidence) error {
 
 // AddEvidenceFromConsensus should be exposed only to the consensus so it can add evidence to the pool
 // directly without the need for verification.
-func (evpool *Pool) AddEvidenceFromConsensus(ev types.Evidence, time time.Time, valSet *types.ValidatorSet) error {
-	var (
-		vals       []*types.Validator
-		totalPower int64
-	)
+func (evpool *Pool) AddEvidenceFromConsensus(voteA, voteB *types.Vote, time time.Time,
+	valSet *types.ValidatorSet) error {
+
+	ev := types.NewDuplicateVoteEvidence(voteA, voteB, time, valSet)
```

Comment:
can't we pass the duplicate vote evidence as a struct instead of recreating it here?

Reply:
yeah I think you're right that that's probably better. It also makes it easier if in the future we decide to add more evidence from consensus